modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/patchedcell/v0.1.0.json

```diff
@@ -93,6 +93,13 @@
               "description": "The compound used in the pipette to label the cell",
```

Comment:
Looks globally good.

I would suggest to bump the version of this schema (experiment/patchedcell/v0.1.0) to v0.2.0 so that users that have already deployed it are not impacted.

I would suggest also to take the opportunity of a new version to update the versions of the imported schemas. The following 'imports' object can be used:

```
{
...
"imports": [
    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
    "{{base}}/schemas/neurosciencegraph/commons/quantitativevalue/v0.1.2",
    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.1",
    "{{base}}/schemas/neurosciencegraph/commons/brainlocation/v0.1.1"
  ],
...
}

The different 'node' values in the schema need to be updated accordingly.

It would be good if @annakristinkaufmann comments on this when she will be back next week. Meanwhile I'll be happy to merge if the above suggestions are taken into account.